### PR TITLE
Game Map-HavenOrbital

### PR DESCRIPTION
**Pull request by MaloTV**
## About the PR
I've added the biggest map yet called Haven Orbital, a map that took me over 600 hours to create and a lot of sweat, tears and frustration, but I finally decided it's done and it's time to PR it again.

Haven Orbital is not just a station, it has a little story that you can see in the video below in the Media field.

Haven Orbital is a 65-80 player map, but I'm pretty sure it could easily hold 200+ players.
It has its own evacuation shuttle called Emergency Bean, also made by me and inspired by the Barratry Shuttle.

- It has a zoo, a reporter's studio and a large green recreation area on the east side of the station.
- Medical checkpoints to support paramedics help the injured 
- Interesting and thoughtful spaces of maintenance to discover

Since the last pull request I have implemented the feedback from the maintainers and co and changed a few things myself that I thought might cause complaints I am confident that all the major problems with the map have been fixed and it is ready to go into the map pool

## Media
**Lore Video**
[https://www.youtube.com/watch?v=PsucLGNMTIc&t](https://www.youtube.com/watch?v=PsucLGNMTIc)

_Full Map Render_
HavenOrbitalPrCompressed

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

:cl:MaloTheDuck
- add: Your crew meets the requirements to inhabit Haven Orbital, the most luxurious station NT has built to date consider yourself lucky!

### CODE REVIEW

**Comment by lzk228** [comment]
Haven my beloved

**Comment by ItsMeThom** [comment]
Absolutely gorgeous map, can't wait to explore it!

image
It seems like these trees might be misaligned, or is that just an artifact of the whole-map screen capture?

**Comment by MaloTV** [comment]
> Absolutely gorgeous map, can't wait to explore it!
> 
> image It seems like these trees might be misaligned, or is that just an artifact of the whole-map screen capture?

Trees become offset on Map Renders they are the right way on the map tho thanks for the nice comment👍

**Comment by TexCruize** [comment]
Hmm yes this map looks like a station 10/10 pls merge

**Comment by UbaserB** [comment]
its a dumb station :3
-10/10 silly malo malo

**Comment by VasilisThePikachu** [comment]
Much big, Very effort.

File: Resources/Prototypes/Maps/Pools/default.yml [hunk 1]
```diff
@@ -15,3 +15,4 @@
   - Origin
   - Saltern
   - Packed
+  - HavenOrbital
```
[the remark this answers is not in the document]
Comment (deltanedas) [reply Resources/Prototypes/Maps/Pools/default.yml:18]: it isnt

**Comment by UbaserB** [comment]
Why did this suddenly get reviewed now

**Comment by Emisse** [comment]
map needs serious layout changes/rebalancing to fit a specific population target. comment for me to reopen if you are gonna get around to it

**Pull request closed without merging**